### PR TITLE
Add default configs copy

### PR DESCRIPTION
**Pull request by sabbaka**

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -10,3 +10,41 @@
   package:
     name: cockpit-session-recording
   when: install_session_player
+
+- name: configure basic sssd
```
Comment (nkinder) [inline tasks/main.yml:18]: Your task name and the actual task performed don't match for this task and the next (they are swapped).

File: tasks/main.yml [hunk 2]
```diff
@@ -10,3 +10,41 @@
...
+    src: 'etc/sssd/conf.d/sssd-session-recording.conf.j2'
+    dest: /etc/sssd/conf.d/sssd-session-recording.conf
+    owner: root
+    group: root
+    mode: 0600
+
+- name: configure sssd session recording config
+  template:
+    src: 'etc/sssd/sssd.conf.j2'
+    dest: /etc/sssd/sssd.conf
```
Comment (nkinder) [inline tasks/main.yml]: We should only create a sssd.conf if one doesn't already exist.  You can do this by providing the `force: no` parameter to the template module.  This parameter defaults to `yes`, which means existing config will be overwritten with your current patch.

Comment (nkinder) [reply tasks/main.yml]: I also believe that we only need to create `/etc/sssd/sssd.conf` on systems that use older versions of SSSD, as current versions will load config snippets from `/etc/sssd/conf.d` even if `/etc/sssd/sssd.conf` doesn't exist.  We should make this task conditional, perhaps based on the value returned by `sssd --version`.  I have an inquiry out to the SSSD developers to find out what version first added this capability.

File: tasks/main.yml [hunk 3]
```diff
@@ -10,3 +10,41 @@
...
+    group: root
+    mode: 0600
+
+# for future purpose. in template use _UID or keep programname if it works
+- name: getent for tlog
+  getent:
+    database: passwd
+    key: tlog
+
+- name: configure rsyslog to elastic logging
```
Comment (nkinder) [inline tasks/main.yml:47]: Let's add a `when` condition to this that only runs this task if a variable like `session_recording_output: elasticsearch` is specified.

File: tasks/main.yml (hunk 2 above)
Comment (nkinder) [reply tasks/main.yml]: The version of SSSD where this snippet issue will be fixed is 2.1.

**Review by nkinder — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 2 above)
Comment (sabbaka) [reply tasks/main.yml]: As I understand we still must provide basic sssd config in order to session recording to work at all. I am not sure about this though. Anyway we could add this to conf.d later

Comment (nkinder) [reply tasks/main.yml]: We will not need the basic SSSD config for our session recording snippet to work in SSSD 2.1 and later (this is not released anywhere yet). We do need it now though.  I have confirmed this with the SSSD developers.

Comment (sabbaka) [reply tasks/main.yml]: Right, there was a but that sssd.conf was required, but will just the session recording config through sssd will be enought for sssd to function at all? As I understood it should work, right?

**Comment by sabbaka** [comment]
I've updated the PR. The sssd.conf could be simplified by using minimum confing from: 
https://pagure.io/SSSD/sssd/issue/3439
https://pagure.io/SSSD/sssd/c/8a3517c5466c107f4d4e0970a1c33b51d6c762f8

These links are in commented lines in tasks/main.yml

**Comment by sabbaka** [comment]
Updated

**Comment by sabbaka** [comment]
I've tested it more or less and I think it is ready to be merged.

**Comment by sabbaka** [comment]
I added services restart and params for sssd

**Pull request merged**